netlify.toml

```diff
@@ -13,14 +13,14 @@
 [[redirects]]
   # Telemetry link
   from = "/telemetry/track"
-  to = "https://api.netlify-services.com/telemetry/track"
-  status = 200
+  to = "https://cli-telemetry.netlify.engineering/.netlify/functions/track"
+  status = 302
```

Comment:
Redirect instead of rewrite seems to work, with the cost of the additional hop. Can you confirm this @JGAntunes?

Reply:
The problem with a `3xx` redirect is that the clients will be changing their request methods from `POST` to `GET`. E.g. [check the logs for the last hour](https://app.netlify.com/sites/cli-telemetry-proxy/functions/track)

Reply:
🤦